pkg/standalone-utils/test/foundry/ProtocolFeeSweeper.t.sol

```diff
@@ -246,6 +246,22 @@ contract ProtocolFeeSweeperTest is BaseVaultTest {
         assertEq(usdc.balanceOf(address(feeRecipient)), DEFAULT_AMOUNT * 2, "USDC not forwarded");
     }
 
+    function testSweepProtocolFeesIfBurnerDoesNotTakeTokens() public {
```

Comment:
```suggestion
    function testSweepProtocolFeesIfBurnerDoesNotPullTokens() public {
```
TakeTokens is confusing to me; it still sounds like the burner is receiving tokens, not *actively* pulling them from the sender. Consider either "PullTokens" as above, or spelled out: "...DoesNotTransferTokensFromSweeper"